docs/meetings/web/2021-09-21.md

```diff
@@ -0,0 +1,36 @@
+# Web frontend meeting 21 September 2021
+
+**Present**: Aapeli, Jesse, Lucas, Natalia, Ricardo
+
+## Discussion
+
+- weblate and i18next is waiting for workspaces
```

Comment:
Did we actually get a conclusion from the investigation/research since the issue is still open?

Reply:
I didn't implement anything, but a little research and recommendations from a few people indicated these are most likely the best options